olive/azureml/azureml_client.py

```diff
@@ -101,6 +101,22 @@ def create_client(self):
                 credential=self._get_credentials(), path=self.aml_config_path, read_timeout=self.read_timeout
             )
 
+    def create_registry_client(self, registry_name: str):
```

Comment:
could we merge it with current create_client? It seems most of the logic is same.

Reply:
I updated this logic. Now registry client doesn't need workspace info.